### ISSUE TITLE
Embree version changes could cause an error in the dockerfile builds

### ISSUE THREAD

**Issue by pshriwise** [pshriwise/double-down#2]
Hi all

Just a preemptive issue notice as it look like embree is moving from version 3.12.1 to 3.12.2

This impact the Dockerfile which currently points to 3.12.1 in the double-down build

openmc/Dockerfile

Line 111 in c262407

 -DEMBREE_DIR=/embree/lib/cmake/embree-3.12.1 ; \ 
I think that when the Dockerfile rebuild is triggered again that we will see a build error (I am seeing this in some other Dockerfile builds)

This can be temporarily fixed by changing

-DEMBREE_DIR=/embree/lib/cmake/embree-3.12.1 ; \
to
-DEMBREE_DIR=/embree/lib/cmake/embree-3.12.2 ; \

This might be a pain to keep up to date so perhaps there are better ideas from those more familiar with the double-down build process @pshriwise

**Comment by Shimwell** [comment]
I think @pshriwise  fixed a little while back and now the ```DEMBREE_DIR``` flag points to the embree folder like this which allows for different version names

```
    cmake .. -DMOAB_DIR=/MOAB \
             -DCMAKE_INSTALL_PREFIX=.. \
             -DEMBREE_DIR=/embree ; \
```

**Comment by pshriwise** [comment]
Yep. All fixed in cf01c4f019b52f3ce1015fc4556cf9e05e0b9c9c.
